### DISCUSSION TITLE
pull-request title with "docs:" like with "[CI:DOCS]"

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [containers/podman#15226] [Ideas]
Maybe it should be understood as a shorthand?

**Comment by vrothberg** [comment]
Are you looking for "docs:" to have the same effect as "[CI:DOCS]"?

Cc: @edsantiago

**Reply by unknowndevQwQ** [reply]
yep

**Reply by edsantiago** [reply]
Why?

**Reply by rhatdan** [reply]
It is easier to type.  :^)
